src/Simulation/QCTraceSimulator/QubitAvailabilityTimeTracker.cs

```diff
@@ -0,0 +1,63 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
```suggestion
﻿// Copyright (c) Microsoft Corporation.
```
If I understand correctly (please confirm!), current guidance is to not include "All rights reserved" in the header for new source files.

Reply:
This is a rename of the file, not a new one. Confirmed that "All rights reserved." should not be included. I will remove it.